src/campaign_server/campaign_server.cpp

```diff
@@ -270,7 +270,7 @@ void server::handle_read_from_fifo(const boost::system::error_code& error, std::
 	std::string cmd;
 	std::getline(is, cmd);
 
-	const control_line ctl = cmd;
+	const control_line ctl (cmd);
```

Comment:
Prefer `type name{args};` to avoid the Most Vexing Parse.

Reply:
I wanted to get back to that. I went with the MVP form becuase it works all the way back to the beginning and I wanted to check when the init-list form appeared. Don't want to use a 17-only feature and I don't keep details like when what appeared in my head. That's what cppreference.com is for.

Reply:
We're using the braced-init-list in other places. It's C++11. It's fine.